README.md

```diff
@@ -1,55 +1,129 @@
+1. The tag assigned to the Category will be displayed under the 'Inherited tag'. All Category products will inherit the tag assigned to the Category.
+
+![Assign tag to category](docs/media/screen-assign-tags.png)
+
+![Inherit tags](docs/media/screen-inherited-tags.png)
+
+### Assign User Group to a Price List Assignment
+
+1. Go to More->Pricing Module->select a Price list Assignment and open the Price list Assignments details;
+1. Under 'Add condition' select 'Shopper profile' and then 'User Group contain';
+1. Enter the User Group name into the correspondent field;
+1. Save the changes.
+
+![Assign user group to pricelist](docs/media/screen-assign-usergroup-to-pricelist.png)
+
+### Assign User Group to Marketing Promotion
+
+1. Go to More-Marketing module->Promotions;
+1. Select a Promotion and open the promotion details blade;
+1. Under 'Add user group' select 'User group contains';
+1. Enter the user group name into the correspondent field;
+1. Save the changes;
+1. The assigned user group will be applied to the selected promotion.
+
+![Assign user group to promotion](docs/media/screen-assign-usergroup-to-promotion.png)
+
+![User group assigned](docs/media/screen-usergroup-assigned-to-promotion.png)
+
+**Important!** Don't forget to rebuild the search indexes for categories and products after tagging process.
 
 <details><summary>Rebuild indexes</summary><p>
 
 ![indexation](https://user-images.githubusercontent.com/7536694/31497248-d12a5efe-af5e-11e7-8938-b39d2b03ad1e.gif) </p></details>
 
-<details><summary>Managing user groups</summary><p>
+### Tags Propagation Policies
+
+#### DownTree propagation policy
+
+When down-tree propagation policy is selected, all products inherit the Tag assigned to the Category.
+In order to apply this type of propagation policy to Catalog categories and products, do the following:
+
+1. Go to Settings->Personalization->General;
+1. Under 'Selected policy for tags propagation' select 'DownTree';
+1. Restart the VC platform.
 
-![user-groups](https://user-images.githubusercontent.com/7536694/31497349-252148ce-af5f-11e7-9c4b-a2c9545fd853.gif) </p></details>
+![Settings](docs/media/screen-settings.png)
 
-##### As the result, products are displayed to the user only from the category that belongs to his `User group`.
+#### UpTree Policy Propagation
 
-<details><summary>Result view</summary><p>
+When the up-tree policy propagation is applied, the Product category will inherit the Product tag, i.e. if a tag assigned to a specific product in the Category, it will be automatically assigned to the Product Category as well.
 
-![result](https://user-images.githubusercontent.com/7536694/31497775-53d994a4-af60-11e7-8544-90ccdf32f82a.gif) </p></details>
+In order to apply this type of propagation policy to Catalog categories and products, do the following:
 
+1. Go to Settings->Personalization->General;
+1. Under 'Selected policy for tags propagation' select 'UpTree';
+1. Restart the VC platform.
```

Comment:
Do we need to duplicate these actions? Maybe it'd be better to write how to set policy and write what every policy means.